src/region/RegionCreator.tsx

```diff
@@ -116,6 +120,19 @@ export default function RegionCreator({ className, onAbort, onStart, onStop }: P
     };
     const updateDraw = (x: number, y: number): void => {
         const [x2, y2] = getPosition(x, y);
+        const { current: x1 } = positionX1Ref;
+        const { current: y1 } = positionY1Ref;
+        const isSmall = x1 === null || y1 === null || (Math.abs(x2 - x1) < MIN_SIZE && Math.abs(y2 - y1) < MIN_SIZE);
```

Comment:
Can we move this to a separate helper function and invoke it inline in the condition?

Reply:
Is it actually possible for `x1` or `y1` to be null in these cases?

Reply:
It's actually not possible, but it's required to check them by TS.